pandas/tests/io/test_parquet.py

```diff
@@ -806,22 +804,10 @@ def test_additional_extension_arrays(self, pa):
                 "c": pd.Series(["a", None, "c"], dtype="string"),
             }
         )
-        if Version(pyarrow.__version__) >= Version("0.16.0"):
-            expected = df
-        else:
-            # de-serialized as plain int / object
-            expected = df.assign(
-                a=df.a.astype("int64"), b=df.b.astype("int64"), c=df.c.astype("object")
-            )
-        check_round_trip(df, pa, expected=expected)
+        check_round_trip(df, pa, expected=df)
```

Comment:
```suggestion
        check_round_trip(df, pa)
```

If it's `df`, it doesn't need to be specified as `expected` (and the same for the case below)